salesforce/accounts/createAccount.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+import sys
+from pathlib import Path
+
+sys.path.append(str(Path(__file__).resolve().parent.parent))
```

Comment:
This is in all the files. Does a comment make sense here? I don't know why this is needed or if this is standard practice.

Reply:
honestly im not sure this is the right way to do this, but python is scary and confusing. it's essentially here because i organized the files into different folders but we are calling the scripts individually, not as a whole python package - so it doesnt know how to find and import the other files without this.